[2022-01-25 06:35:59,374] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-13T00:00:00+00:00 [queued]>
[2022-01-25 06:35:59,384] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-13T00:00:00+00:00 [queued]>
[2022-01-25 06:35:59,384] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 06:35:59,384] {taskinstance.py:1239} INFO - Starting attempt 1 of 6
[2022-01-25 06:35:59,384] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 06:35:59,397] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): first_task> on 2022-01-13 00:00:00+00:00
[2022-01-25 06:35:59,403] {standard_task_runner.py:52} INFO - Started process 3876 to run task
[2022-01-25 06:35:59,407] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'this_is_first_dag', 'first_task', 'scheduled__2022-01-13T00:00:00+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp95666r7p', '--error-file', '/tmp/tmp751ilgsv']
[2022-01-25 06:35:59,411] {standard_task_runner.py:77} INFO - Job 2: Subtask first_task
[2022-01-25 06:35:59,472] {logging_mixin.py:109} INFO - Running <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-13T00:00:00+00:00 [running]> on host ceab856ace11
[2022-01-25 06:35:59,651] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflaw_test
AIRFLOW_CTX_DAG_ID=this_is_first_dag
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-13T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-13T00:00:00+00:00
[2022-01-25 06:35:59,652] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-25 06:35:59,652] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo Hello, this is my first dag. Are you OK?']
[2022-01-25 06:35:59,666] {subprocess.py:85} INFO - Output:
[2022-01-25 06:35:59,667] {subprocess.py:89} INFO - Hello, this is my first dag. Are you OK?
[2022-01-25 06:35:59,667] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-25 06:35:59,702] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=this_is_first_dag, task_id=first_task, execution_date=20220113T000000, start_date=20220125T063559, end_date=20220125T063559
[2022-01-25 06:35:59,740] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-25 06:35:59,868] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-01-25 07:27:42,650] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-13T00:00:00+00:00 [queued]>
[2022-01-25 07:27:42,662] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-13T00:00:00+00:00 [queued]>
[2022-01-25 07:27:42,662] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 07:27:42,662] {taskinstance.py:1239} INFO - Starting attempt 1 of 6
[2022-01-25 07:27:42,662] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 07:27:42,673] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): first_task> on 2022-01-13 00:00:00+00:00
[2022-01-25 07:27:42,678] {standard_task_runner.py:52} INFO - Started process 334 to run task
[2022-01-25 07:27:42,682] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'this_is_first_dag', 'first_task', 'scheduled__2022-01-13T00:00:00+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpevqqd3xx', '--error-file', '/tmp/tmptoo0nquc']
[2022-01-25 07:27:42,686] {standard_task_runner.py:77} INFO - Job 2: Subtask first_task
[2022-01-25 07:27:42,742] {logging_mixin.py:109} INFO - Running <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-13T00:00:00+00:00 [running]> on host 943a4c094194
[2022-01-25 07:27:42,916] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflaw_test
AIRFLOW_CTX_DAG_ID=this_is_first_dag
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-13T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-13T00:00:00+00:00
[2022-01-25 07:27:42,917] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-25 07:27:42,917] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo Hello, this is my first dag. Are you OK?']
[2022-01-25 07:27:42,929] {subprocess.py:85} INFO - Output:
[2022-01-25 07:27:42,931] {subprocess.py:89} INFO - Hello, this is my first dag. Are you OK?
[2022-01-25 07:27:42,931] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-25 07:27:42,957] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=this_is_first_dag, task_id=first_task, execution_date=20220113T000000, start_date=20220125T072742, end_date=20220125T072742
[2022-01-25 07:27:43,014] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-25 07:27:43,089] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
